lib/iron_trail/testing/rspec.rb

```diff
@@ -1,3 +1,44 @@
 # frozen_string_literal: true
 
 require 'iron_trail'
+
+module IronTrail
+  module Testing
+    class << self
+      attr_accessor :enabled
+
+      def enable!
+        DbFunctions.new(ActiveRecord::Base.connection).install_functions
+        @enabled = true
+      end
+
+      def disable!
```

Comment:
It's unlikely to happen since you need to require this, but one might do it in a rails console, I'd like iron_trail to protect me from doing this in production

Reply:
agreed. I'll add a protection if `ENV['RAILS_ENV'] == 'production'`